make/validate.mk

```
@@ -8,4 +8,4 @@ $(DKP_BLOODHOUND_BIN):
 
 .PHONY: validate-manifests
 validate-manifests: $(DKP_BLOODHOUND_BIN)
-	$(DKP_BLOODHOUND_BIN) . --skip-applications=kaptain,grafana-loki,project-grafana-loki,project-logging
+	$(DKP_BLOODHOUND_BIN)
```

(nit) - its fine to make CI verbose.
```suggestion
	$(DKP_BLOODHOUND_BIN) -v 4
```

---

In this case it lists all checked resources, which makes it pretty hard to find the failing ones (while not providing much value IMO).